frigate_beta/config.yaml

```diff
@@ -1,8 +1,7 @@
 name: Frigate Beta
-version: 0.14.1
+version: 0.15.0-beta1
 breaking_versions:
-  - 0.14.0
-  - 0.14.1
```

Comment:
We might still want to leave these or someone on older version will auto update

Reply:
I noticed we didn't do it for 0.14.0-beta1, and figured because this was not a "stable" version, we'd be fine. Happy to make the change though